source/manual/screens.html.md

```diff
@@ -28,7 +28,7 @@ There are two screens by the Technical 2nd Line desks.
 The top screen is a webpage running [frame splits][frame-splits] with three splits: [production health, open Zendesk
 tickets and Icinga alert summary per environment][2ndline-dash].
 
-The bottom screen is a PaaS-hosted Grafana dashboard showing statistics for data.gov.uk.
+The bottom screen is a Grafana dashboard showing statistics for data.gov.uk.
```

Comment:
Do we even have a DGU Grafana dashboard on that second screen anymore? 🤔 This whole page could probably do with an update.
But if we don't have a DGU Grafana then we should at least drop this sentence, and consider cropping it out of the monitoring.jpg image.